appveyor.yml

```diff
@@ -3,6 +3,10 @@ pull_requests:
   do_not_increment_build_number: true
 skip_tags: true
 skip_branch_with_pr: false
```

Comment:
Also it seems that it should be `skip_branch_with_pr: true` . @joemcbride  ?

Reply:
Seems like there was a reason for having that false, but I can double check.